demo/context-menu-basic-demos.html

```diff
@@ -6,6 +6,52 @@
       }
     </style>
 
+    <h3>Nested Menu Items</h3>
+    <vaadin-demo-snippet id="context-menu-items-demo">
```

Comment:
Consider moving to separate demo page. 
BTW  it would be also add `when-defined="vaadin-context-menu"` for demo snippet.

Reply:
Done.